apps/linode-marketplace-nats-single-node/roles/nats/templates/nats.service.j2

```diff
@@ -0,0 +1,16 @@
+[Unit]
+Description=NATS Server
+After=network.target ntp.service
+[Service]
+PrivateTmp=true
+Type=simple
+ExecStart=/usr/local/bin/nats-server -c /etc/nats.conf
+ExecReload=/bin/kill -s HUP $MAINPID
+ExecStop=/bin/kill -s SIGINT $MAINPID
+Restart=on-failure
+RestartSec=5s
+KillMode=process
+User=root
```

Comment:
Should NATs have it's own system user instead of being owned by root?

Reply:
Added nats user and required changes given we are not using root.